### PR TITLE
Pr209

### PR DESCRIPTION
**Pull request by AndreyKozlov1984**
#209 with fixes

### CODE REVIEW

File: cached_logos/cilium.svg [hunk 1]
```diff
@@ -1 +1,5 @@
...
+>>>>>>> origin/main
```
Comment (mohammed90) [inline cached_logos/cilium.svg:5]: Beware, this file and a few others contain the merge conflict markers.

Comment (AndreyKozlov1984) [reply cached_logos/cilium.svg]: @mohammed90 fixed!

File: cached_logos/caddy.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...1489.76 -444.051)"/></g></g></g><g id="R"><g><g><path fill="#2e2e2e" fill-rule="nonzero" d="M0.073,-0L0.073,-0.7L0.383,-0.7C0.428,-0.7 0.469,-0.69 0.506,-0.67C0.543,-0.651 0.572,-0.623 0.594,-0.588C0.616,-0.553 0.627,-0.512 0.627,-0.465C0.627,-0.418 0.615,-0.377 0.592,-0.342C0.569,-0.306 0.539,-0.279 0.501,-0.259L0.57,-0.128C0.574,-0.12 0.579,-0.115 0.584,-0.111C0.59,-0.107 0.596,-0.106 0.605,-0.106L0.664,-0.106L0.664,-0L0.587,-0C0.56,-0 0.535,-0.007 0.514,-0.02C0.493,-0.034 0.476,-0.052 0.463,-0.075L0.381,-0.232C0.375,-0.231 0.368,-0.231 0.361,-0.231C0.354,-0.231 0.347,-0.231 0.34,-0.231L0.192,-0.231L0.192,-0L0.073,-0ZM0.192,-0.336L0.368,-0.336C0.394,-0.336 0.417,-0.341 0.438,-0.351C0.459,-0.361 0.476,-0.376 0.489,-0.396C0.501,-0.415 0.507,-0.438 0.507,-0.465C0.507,-0.492 0.501,-0.516 0.488,-0.535C0.475,-0.554 0.459,-0.569 0.438,-0.579C0.417,-0.59 0.394,-0.595 0.369,-0.595L0.192,-0.595L0.192,-0.336Z" transform="translate(4310.105 1782.567) scale(6.02617)"/></g></g><g transform="translate(2390.518 823.203) scale(.49702)"><circle cx="3866.43" cy="1926.14" r="8.923" fill="none" stroke="#2e2e2e" stroke-linecap="butt" stroke-linejoin="miter" stroke-width="2"/></g></g></g></g></g></svg>
```
[the remark this answers is not in the document]
Comment (AndreyKozlov1984) [reply cached_logos/caddy.svg:1]: files in cached_logos are autogenerated, but regarding hosted_logos - all I did was merging the main branch into it.

[the remark this answers is not in the document]
Comment (AndreyKozlov1984) [reply cached_logos/caddy.svg:1]: the file name is based on an item name, not on a filename in a hosted_logos folder

[the remark this answers is not in the document]
Comment (mohammed90) [reply cached_logos/caddy.svg:1]: Noted. Then I made a mistake when I manually placed a file in `cached_logos`. Thanks 👍🏼

**Comment by tracymiranda** [comment]
LGTM! Thanks all!

**Pull request merged**